src/EventLoop.hs

```diff
@@ -57,6 +57,10 @@ eventFromPullRequestPayload payload =
   in
     case payload.action of
       Github.Opened      -> Logic.PullRequestOpened number branch baseBranch sha title author body
+      -- If the pull request is reopened, we do not want to parse the command from
+      -- the body, because it means that a person intefered with the pull request.
+      -- This is why we do not pass the body to the request, as this means that we
+      -- will not look at a possible command in the body.
```

Comment:
```suggestion
      -- If the pull request is reopened, we do not want to parse the command from
      -- the body, because it means that a person manually intervened with the pull 
      -- request. This is why we do not pass the body to the even, so we don't 
      -- accidentally repeatedly process the same command.
```